src/processors/default/createNftFromCandyMachine.ts

```diff
@@ -0,0 +1,52 @@
+
+import * as MetaplexFoundation from '@metaplex-foundation/js';
+import { ConfirmedSignatureInfo } from '@solana/web3.js';
+
+import { Table } from '../../db/db';
+import { newCandyMachineNfts } from '../../triggers/newCandyMachineNfts';
+import { MetaFactory, MetaFactoryTypes } from '../../types/metaFactory';
+import { NFT, NftFactory } from '../../types/nft';
+import { Clients, Processor } from '../../types/processor';
+
+
+export const createNftsFromCandyMachine: (metaFactory: MetaFactory) => Processor = (metaFactory) => {
+
+  return {
+    name: 'createNftsFromCandyMachine',
+    trigger: newCandyMachineNfts(metaFactory),
+
+    processorFunction: async (items: { metadataAccount: MetaplexFoundation.Metadata, mintTx: ConfirmedSignatureInfo }[], clients: Clients) => {
+
+      const nfts = items.map(({ metadataAccount, mintTx }) => {
+        const mintAddress = metadataAccount.mintAddress.toBase58();
+
+        const details: Omit<NFT, 'owner' | 'tokenId' | 'standard'> = {
+          id: mintAddress,
+          contractAddress: mintAddress,
+          platformId: metaFactory.platformId,
+          tokenMetadataURI: metadataAccount!.uri,
+          tokenURI: metadataAccount!.uri,
+          approved: true,
```

Comment:
approved by default

Reply:
should we use `metaFactory.autoApprove` instead ?